java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -426,3 +709,44 @@ predicate containerStep(Expr n1, Expr n2) {
   containerReturnValueStep(n1, n2) or
   containerUpdateStep(n1, n2)
 }
+
+predicate arrayStoreStep(Node node1, Node node2) {
```

Comment:
Make private or document

Reply:
Hmm, why wasn't this caught by the PR check?